coding/compressed_bit_vector.cpp

```diff
@@ -185,6 +185,7 @@ DenseCBV::DenseCBV(vector<uint64_t> const & setBits)
 {
   if (setBits.empty())
   {
+    m_popCount = 0;
```

Comment:
Why? You've specified in header that m_popCount should be initialized to zero each time new instance is instantiated. Initializations in header are not obvious, so either remove this initialization to zero or remove zero-initialization from header.